common/lib/capa/capa/xqueue_interface.py

```diff
@@ -15,6 +15,8 @@
 
 # Wait time for response from Xqueue.
 XQUEUE_TIMEOUT = 35  # seconds
+CONNECT_TIMEOUT = 3.05  # seconds
```

Comment:
the naming conventions of python for timeout is also 'connect' and 'read' so we should follow that convention also!